robottelo/ui/activationkey.py

```diff
@@ -131,16 +131,11 @@ def update(self, name, new_name=None, description=None,
 
     def delete(self, name, really=True):
         """Deletes an existing activation key."""
-        element = self.search(name)
-
-        if element:
-            element.click()
-            self.wait_for_ajax()
-            self.click(locators['ak.remove'])
-            if really:
-                self.click(common_locators['confirm_remove'])
-            else:
-                self.click(locators['ak.cancel'])
+        self.delete_entity(
```

Comment:
I thought that step is out of scope of current issue as per comment above. We have to remove that dependency, but that will require adding few more variables. I don't mind to work on that after that PR